### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Dannie226** [RedstoneWizard08/QuickHull#1]
In the documentation, I feel as though it is important to note that the create hull function only returns an array of faces. You are responsible for storing the points somewhere for creating the hull for rendering or other purposes. Or, that could be an improvement for the future. That's for you to decide.

**Comment by RedstoneWizard08** [comment]
Actually, I was going to ask, can you help me make the documentation? I'm going to start off with a TypeDoc-generated documentation website, but could you help me make it better, @Dannie226?

**Comment by Dannie226** [comment]
Yeah. I can do that. Just generate the docs, and I can go through them in my free time and make sure that everything looks correct. Is the repo public, or will you need to add me as a contributor for the repo?

**Comment by RedstoneWizard08** [comment]
It's public, but I'm going to add you as a contributor as well, since you're going to help with the docs and you're the original math genius behind all this.

**Comment by Dannie226** [comment]
Umm... This was just an implementation of the quick hull algorithm originally made by someone else. I took the file, put everything necessary in one file, and created the utils functions so the hull could be created, and I added the convenience method. The actual quickhull math/implementation was done by someone else. I just made it such that you only needed to include one file and added a method for creating a hull very easily.

**Comment by RedstoneWizard08** [comment]
Oh ok lol

**Comment by Dannie226** [comment]
Could you also put the repo under an MIT license? It's part of the original, and I, for one, don't wish to get sued.

**Comment by RedstoneWizard08** [comment]
Okay

**Comment by RedstoneWizard08** [comment]
What name should I put it under?

**Comment by Dannie226** [comment]
You are the creator of the repo, but if you wish to attribute me, you can

**Comment by RedstoneWizard08** [comment]
Okay, I added docs and created an action to deploy them

https://github.com/RedstoneWizard08/QuickHull/runs/6329959684?check_suite_focus=true

**Comment by RedstoneWizard08** [comment]
https://redstonewizard08.github.io/QuickHull

**Comment by Dannie226** [comment]
With the auto-generated docs, are we able to add descriptions for everything?

**Comment by RedstoneWizard08** [comment]
Yes, using JSDoc comments.

**Comment by Dannie226** [comment]
Aight. I'll get started on adding those after I refresh myself on the documentation.

**Comment by Dannie226** [comment]
I only got the util functions done, but that look right?

**Comment by RedstoneWizard08** [comment]
Lemme check - sorry I was in a piano lesson xD

**Comment by RedstoneWizard08** [comment]
That looks right, but since it's typescript, you don't need the type parameters in the JSDoc, so you can change this:

```diff
- * @param {number[]} a
+ * @param a
```
